netconf.c

```diff
@@ -1213,42 +1423,53 @@ handle_kill_session (struct netconf_session *session, xmlNode * rpc)
     xmlNode *action = xmlFirstElementChild (rpc);
     xmlNode *node;
     uint32_t kill_session_id = 0;
-    char *msg = NULL;
     struct netconf_session *kill_session = NULL;
     xmlChar* content = NULL;
+    nc_error_parms error_parms = NC_ERROR_PARMS_INIT;
```

Comment:
I think you might need to be careful about cleanup for all these initialised error structures? Have you run apteryx-netconf tests on a PC with valgrind running?